pkgs/tools/text/reckon/default.nix

```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
   in ''
     runHook preInstall
     mkdir -p $out/bin
-    makeWrapper ${env}/bin/reckon $out/bin/reckon
+    makeWrapper ${env}/bin/reckon $out/bin/reckon  --set PATH ${lib.makeBinPath [ file ]}
```

Comment:
We don't want to overwrite PATH but append to it. Also please add a line break with \